config/routes.rb

```diff
@@ -1,6 +1,5 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  resources :mechanics, only: [:show, :update]
```

Comment:
SInce we are not updating a specific mechanic and instead creating a new relationship between a mechanic and a ride, you would not have a `:update` route for a mechanic here. Here is what your routes should look like:

```ruby
resources :mechanics, only: [:show] do
  resources :rides, only: [:create], controller: :mechanic_rides
end

resources :amusement_parks, only: [:show]
```